phpBB/includes/acp/acp_extensions.php

```diff
@@ -1130,4 +1126,14 @@ public function output_metadata_to_template($metadata)
 			));
 		}
 	}
+
+	protected function check_is_enableable($enableable)
```

Comment:
Please add a docblock to this.

Reply:
Just a thought... Another approach (which I think is may be more cognitive/readable, could be like):
```php
$extension = $this->ext_manager->get_extension($ext_name);
if ($message = $extension->is_enableable() !== true)
{
	$this->show_error($message);
}

/**
* Trigger an error message for an extension that could not be enabled.
* Error message can be set by the extension.
*
* @param bool|array $message Array of reasons (set by the extension) 
* or false for default reason.
*/
public function show_error($message)
{
	if (is_array($message))
	{
		$message = implode('<br />', $message)
	}
	else if (empty($message)
	{
		$message = $this->user->lang('EXTENSION_NOT_ENABLEABLE');
	}

	trigger_error($message . adm_back_link($this->u_action), E_USER_WARNING);
}
```